recipes/genenotebook/build.sh

```diff
@@ -7,27 +7,29 @@ mkdir -p $outdir $PREFIX/bin
 pwd
 ls -lah .
 
-CC --version || echo 'cc not found'
-GCC --version || echo 'gcc not found'
-
-git clone https://github.com/meteor/meteor
-
-pushd meteor
-git checkout 39cc07472f8f1dc820b29b8c5a30813051d62700 #V1.9.0 commit
-
-
-NODE_FROM_SRC='' ./scripts/build-node-for-dev-bundle.sh
-
-./scripts/generate-dev-bundle.sh
-
-./meteor --version
-export PATH=$(pwd):$PATH
-popd
-
+#CC --version || echo 'cc not found'
+#GCC --version || echo 'gcc not found'
+#
+#git clone https://github.com/meteor/meteor
+#
+#pushd meteor
+#git checkout 39cc07472f8f1dc820b29b8c5a30813051d62700 #V1.9.0 commit
+#
+#
+#NODE_FROM_SRC='' ./scripts/build-node-for-dev-bundle.sh
+#
+#./scripts/generate-dev-bundle.sh
+#
+#./meteor --version
+#export PATH=$(pwd):$PATH
+#popd
+#
 npm install --unsafe-perm
-ls -lah .
 
-meteor node -v
+export PATH=$PATH:"$HOME/.meteor"
```

Comment:
I'm not sure if using $HOME here will work, maybe it would be better to use the `-g` option on `npm install`?

Reply:
Those would have a different effect, meteor is installed in `$HOME/.meteor` (the paths is not configurable) and is used for subsequent building of the code. Anyway, the builds pass so I don't think this has to be changed?

Reply:
Ah ok, you mean it's only used while building? Because "$HOME/.meteor" is not in the final package archive (which is fine if not used at runtime)

Reply:
Indeed!

Reply:
Ok great, so I can merge if it looks good for you

Reply:
lgtm!